tests/testthat/test-helpers.R

```diff
@@ -188,16 +188,27 @@ test_that("h_get_slots throws the error for non-existing slots", {
 # h_join_models ----
 
 test_that("h_join_models works as expected", {
-  model1 <- function(x) { x <- x - 2; x <- x^2 } # nolintr
-  model2 <- function(x) { x^3 } # nolintr
+  model1 <- function(x) {
+    x <- x - 2
+    x <- x^2
+  } # nolintr
```

Comment:
I guess we no longer need `nolintr` here - generally we try to have as few as possible of them.